include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -63,10 +64,30 @@ using __has_known_identity =
 #else //_USE_GROUP_ALGOS && defined(SYCL_IMPLEMENTATION_INTEL)
 
 template <typename _BinaryOp, typename _Tp>
-using __has_known_identity = std::false_type;
+using __has_known_identity_src = std::false_type;
 
 #endif //_USE_GROUP_ALGOS && defined(SYCL_IMPLEMENTATION_INTEL)
 
+template <typename _Tp>
+using __able_to_use_known_identity_src =
+#if !defined(_ONEDPL_ICPX_USE_KNOWN_IDENTITY_FOR_ARITHMETIC_64BIT_DATA_TYPES) ||                                       \
```

Comment:
I believe `!defined(_ONEDPL_ICPX_USE_KNOWN_IDENTITY_FOR_ARITHMETIC_64BIT_DATA_TYPES)` can be removed since `_ONEDPL_ICPX_USE_KNOWN_IDENTITY_FOR_ARITHMETIC_64BIT_DATA_TYPES` will evaluate to 0 if undefined.

Reply:
@mmichel11 disagree with you.